chrome/content/scaffold/scaffold.js

```diff
@@ -1780,7 +1790,7 @@ var Scaffold = new function () {
 	 * the system's default browser).
 	 * @param {boolean} openExternally whether to open in the default browser
 	**/
-	this.openURL = function (openExternally) {
+	this.openURLInTests = function (openExternally) {
```

Comment:
"openURLInTests" is a little confusing. Something like `openURLFromSelectedTest()` and `openURLFromURLField()` would be better.